acceptance/managers/input_configuration_manager.go

```diff
@@ -0,0 +1,91 @@
+// +build acceptance
+
+package managers
+
+import (
+	"encoding/json"
+	"os"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+)
+
+const (
+	envPackPath                 = "PACK_PATH"
+	envPreviousPackPath         = "PREVIOUS_PACK_PATH"
+	envPreviousPackFixturesPath = "PREVIOUS_PACK_FIXTURES_PATH"
+	envLifecyclePath            = "LIFECYCLE_PATH"
+	envPreviousLifecyclePath    = "PREVIOUS_LIFECYCLE_PATH"
+	envGitHubToken              = "GITHUB_TOKEN"
+	envAcceptanceSuiteConfig    = "ACCEPTANCE_SUITE_CONFIG"
+	envCompilePackWithVersion   = "COMPILE_PACK_WITH_VERSION"
+)
+
+type InputConfigurationManager struct {
+	packPath                 string
+	previousPackPath         string
+	previousPackFixturesPath string
+	lifecyclePath            string
+	previousLifecyclePath    string
+	compilePackWithVersion   string
+	githubToken              string
+	combinations             ComboSet
+}
+
+func NewInputConfigurationManager() (InputConfigurationManager, error) {
+	packPath := os.Getenv(envPackPath)
+	previousPackPath := os.Getenv(envPreviousPackPath)
+	previousPackFixturesPath := os.Getenv(envPreviousPackFixturesPath)
+	lifecyclePath := os.Getenv(envLifecyclePath)
+	previousLifecyclePath := os.Getenv(envPreviousLifecyclePath)
+	compilePackWithVersion := os.Getenv(envCompilePackWithVersion)
+	githubToken := os.Getenv(envGitHubToken)
+
+	err := absolutizePaths(&packPath, &previousPackPath, &previousPackFixturesPath, &lifecyclePath, &previousLifecyclePath)
```

Comment:
Didn't realize absolutize is a word! https://www.merriam-webster.com/dictionary/absolutize

Reply:
Just be glad I didn't try and colonise https://en.wiktionary.org/wiki/absolutise

Reply:
Haha @dfreilich great minds think alike: https://github.com/buildpacks/pack/pull/708#discussion_r445182334 :)